test-utils/state-viewer/src/state_dump.rs

```diff
@@ -0,0 +1,201 @@
+use near_chain::RuntimeAdapter;
+use near_chain_configs::{Genesis, GenesisConfig};
+use near_primitives::block::BlockHeader;
+use near_primitives::state_record::StateRecord;
+use near_primitives::types::{AccountInfo, StateRoot};
+use near_store::TrieIterator;
+use neard::NightshadeRuntime;
+use std::collections::HashMap;
+
+pub fn state_dump(
+    runtime: NightshadeRuntime,
+    state_roots: Vec<StateRoot>,
+    last_block_header: BlockHeader,
+    genesis_config: &GenesisConfig,
+) -> Genesis {
+    println!("Generating genesis from state data");
```

Comment:
`"Generating genesis from state data of #{} / {}", last_block_header.inner_lite.height`

Reply:
what is the second argument?

Reply:
`block_hash`